contracts/lib/SignatureVerification.sol

```diff
@@ -36,6 +33,31 @@ contract SignatureVerification is SignatureVerificationErrors, LowLevelHelpers {
         bytes32 digest,
         bytes memory signature
     ) internal view {
+        address recoveredSigner = _tryEcRecoverSignature(digest, signature);
```

Comment:
I think it's better to add a check for `0` here. A bit worried if `signer` can be `0` for the call.